src/Migrations/Version20250520170950.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20250520170950 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Remove plan to claim GA for one charity';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql(<<<'SQL'
+            UPDATE Donation 
+            SET matchbot.Donation.tbgShouldProcessGiftAid = 0
```

Comment:
```suggestion
            SET tbgShouldProcessGiftAid = 0
```
Possibly needed as with other recent migration – pMA simulate certainly gets confused with the prefix.